app/code/Magento/InventoryCatalogAdminUi/view/adminhtml/web/template/stock/assign-sources/action-delete.html

```diff
@@ -5,7 +5,7 @@
  */
 -->
 <button class="action-delete"
-        attr="{'data-action': 'remove_row', title: $parent.deleteButtonLabel}"
+        attr="{'data-action': 'remove_row', title: $t('Unassign')}"
```

Comment:
`$parent.deleteButtonLabel` may be configured in the `product_form.xml`.

It's best to configure it on that level to avoid hardcoding

Reply:
This is the new output after making the requested changes:

![image](https://user-images.githubusercontent.com/39999239/55668153-d6313100-5865-11e9-852c-bf059e8f94f1.png)